client/src/containers/Profile/Schema.js

```diff
@@ -0,0 +1,23 @@
+import * as Yup from 'yup';
+
+export const inputs = [
```

Comment:
there is no type as phone number, but in schema you are validating the phone no. Any reason?